editors/code/README.md

```diff
@@ -2,8 +2,22 @@
 
 Provides support for rust-analyzer: novel LSP server for the Rust programming language.
 
+**Note** the extension may cause conflicts with the official Rust extension. It is recommended to disable the Rust extension when using the rust-analyzer extension.
 
-Features:
+**Note** the project is in alpha status: it is already useful in practice, but can't be considered stable.
+
+## Sponsor
+
+Work on rust-analyzer is sponsored by
+
+[<img src="https://user-images.githubusercontent.com/1711539/58105231-cf306900-7bee-11e9-83d8-9f1102e59d29.png" alt="Ferrous Systems" width="300">](https://ferrous-systems.com/)
+- [Mozilla](https://www.mozilla.org/en-US/)
+- [Embark Studios](https://embark-studios.com/)
+- [freiheit.com](https://www.freiheit.com)
+
+
```

Comment:
Let's add "Sponsor rust-analyzer on Open Collectieve" while we are at it (see the changelog for lings)

Reply:
done